libs/dataRequests.tsx

```diff
@@ -1,17 +1,50 @@
 "use server";
+import moment from "moment";
 import { redirect } from "next/navigation";
 
 const USERNAME = process.env.NEXT_PRIVATE_USERNAME;
 const PASSWORD = process.env.NEXT_PRIVATE_PASSWORD;
 const BACKEND_URL = process.env.NEXT_PUBLIC_BACKEND_URL;
 
+const backendUrlSplit = BACKEND_URL?.split("://");
+const scheme = backendUrlSplit?.[0];
+const domain = backendUrlSplit?.[1];
+
+const logInfo = (message: string, method: string, url: string) => {
+  console.info({
+    event: message,
+    http: {
+      method: method,
+      scheme: scheme,
+      host: domain,
+      // port: // TODO add port number
```

Comment:
TODO needs doing 😄 , there's a few of these.

Reply:
Added a `BACKEND_PORT` env var as a means to get the port as unable to do it frontside. So will need to be added at some point. Currently defaults to blank.